#Obstaculo formado de uno o mas puntos detectados con la camara d435
PuntosObstaculo[] array_puntos



